contracts/modules/capital/Pool.sol

```diff
@@ -137,7 +140,14 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
         continue;
```

Comment:
it might be a good idea to assign `assets[i]` to a variable so we only SLOAD once